ISBN/.gitignore

```diff
@@ -1,11 +1,3 @@
-src/a/autoload.php
-src/b/autoload.php
-src/c/autoload.php
-src/d/autoload.php
-src/f/autoload.php
-src/g/autoload.php
-src/autoload.php
 build.xml
 build/*
 phpunit.xml
-tests/autoload.php
```

Comment:
autoload dateien gehören grundsätzlich ins .gitignore damit sie nicht im git landen (wie in diesem PR).
schau dir mal https://git-scm.com/docs/gitignore an, es gibt funktionen für platzhalter, sodass man alle autoload.php mit einer zeile ignorieren kann.